src/include/sof/trace/trace.h

```diff
@@ -535,7 +533,7 @@ struct tr_ctx {
 static inline void mtrace_printf(int log_level, const char *format_str, ...)
 {
 };
-
+#endif //MODULE_PRIVAT
```

Comment:
wow... In the previous commit you remove inclusion of many of these headers (and some others) and here you heavily patch them with `#ifdef`s...